### PR TITLE
fix big amounts mess and move formatFinancialData to utils

### PR DESCRIPTION
**Pull request by t0rbik**
**Before**
Screenshot 2023-07-19 at 2 05 49 pm
**After**
Screenshot 2023-07-19 at 2 05 29 pm

---

### Detailed summary
- Added import statement for `formatFinancialData` function in `info.tsx`
- Moved `formatFinancialData` function from `utils.ts` to `info.tsx` and exported it
- Updated `formatFinancialData` function in `utils.ts` to handle larger numbers with suffixes (e.g., "m" for million, "b" for billion)

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by oxSaturn** [comment]
I'm wondering if we should add a white space between the number and symbol, like:

```
3.1415b TOKEN
```

File: utils/utils.ts [hunk 1]
```diff
@@ -13,12 +13,29 @@ export function formatCurrency(amount: any, decimals = 2) {
       maximumFractionDigits: decimals,
     });
 
+    if (amount > 1_000_000_000) {
```
Comment (oxSaturn) [inline utils/utils.ts:16]:
```suggestion
    if (amount >= 1_000_000_000) {
```
Shall we check equal as well?

**Comment by t0rbik** [comment]
I reckon there was no immediate request to fix space from ceazor as with just one multi token

**Review by oxSaturn — APPROVED** [review]

**Comment by t0rbik** [comment]
tbh space is precious and i think we can live without space there for now

**Pull request merged**